### DISCUSSION TITLE
Codespaces Issue: "Git: command 'github.codespaces.publish' not found"

### DISCUSSION THREAD

**Discussion by thecherrie** [community/community#67031] [Codespaces]
### Select Topic Area

Bug

### Body

I'm trying to publish my codespaces branch to a private repository, but after clicking publish branch, the following error is shown:

```
Git: command 'github.codespaces.publish' not found
```

<img width="510" alt="image" src="https://github.com/community/community/assets/14931384/627bd7ed-30e4-4bc0-958d-40e313143f0c">

<img width="291" alt="image" src="https://github.com/community/community/assets/14931384/74a39ea9-77e3-44ae-81c9-a20a0863cf2c">

I can't seem to find anything on the internet on this, so I am making this. 

The template I used for this codespace was (https://github.com/theodorusclarence/ts-nextjs-tailwind-starter)

Any ideas? Thank you

**Comment by vanarok** [comment]
I have also this problem
